packages/web-app-files/src/views/spaces/DriveResolver.vue

```diff
@@ -52,9 +69,47 @@ export default defineComponent({
       }
     }
 
+    const resolveToInternalLocation = async (path: string) => {
+      const internalSpace = getInternalSpace(unref(fileId).split('!')[0])
+      if (internalSpace) {
+        const resource = await clientService.webdav.getFileInfo(internalSpace, { path })
+
+        const resourceId = resource.type !== 'folder' ? resource.parentFolderId : resource.fileId
```

Comment:
Can you explain this a little? Why is the parent folder of the space relevant?

Reply:
Sure, it's relevant because in case of a single file, we want to resolve its parent folder/space and mark the file in the filelist.